blog/15-02-2023-how-to-tokenize-physical-assets-on-celo/how-to-tokenize-physical-assets-on-celo.md

```diff
@@ -0,0 +1,300 @@
+---
+title: How to Tokenize a Physical Asset on Celo
+description: Understand how to start the tokenization on Celo
+authors:
+- name: Ewerton Lopes
+  title: Founder of Guizo Studios
+  url: https://github.com/guizostudios
+  image_url: https://avatars.githubusercontent.com/u/97633823
+tags: [celo, solidity, developers, celosage]
+hide_table_of_contents: false
+slug: /tutorials/how-to-tokenize-a-physical-asset-on-celo
+---
+
+
+![header](../../src/data-tutorials/showcase/beginner/how-to-tokenize.png)
+
+## What is tokenization?
+
+Tokenization is the process of creating a digital token that represents a real-world asset. This token can be traded on a blockchain, which is a distributed ledger that allows for the secure and transparent transfer of ownership of the asset.
+
+Tokenization has the **potential to revolutionize** the way we own and trade assets by making it easier to buy, sell, and transfer ownership of a wide range of assets, including real estate, precious metals, and even fine art. It can also help to increase liquidity and reduce the costs associated with trading assets, as the use of smart contracts and blockchain technology can streamline the transaction process.
+
+On July 14, 2022, Itaú, a big bank in Brazil, launched its own cryptocurrency unit, which aims to tokenize fixed income securities and shares traded in the market in order to make alternatives that are currently restricted to high net worth investors more accessible.
+
+In addition, in the second half of July, the educational platform Pravaler became the first company in the sector to issue tokenized debentures in the local market, raising around U$ 12 million in the operation.
+
+In the Brazil market, there are already companies that have been operating in the area of tokenization of real assets for some years, converting them into NFTs, which stands for Non-Fungible Token, such as MB Tokens, a subsidiary of Mercado Bitcoin focused on the segment, and Hurst Capital.
```

Comment:
In the Brazil market, there are already companies that have been operating in the area of tokenization of real assets for some years, converting them into NFTs, which stands for Non-Fungible Token, such as MB Tokens, a subsidiary of Mercado Bitcoin focused on the segment and Hurst Capital.

Reply:
Done